rmw_opendds_cpp/include/rmw_opendds_cpp/DDSSubscriber.hpp

```diff
@@ -73,14 +76,15 @@ class DDSSubscriber : public DDSEntity
   // Remap the specific OpenDDS DataReader status to a generic RMW status
   rmw_ret_t get_status(const DDS::StatusMask mask, void * rmw_status) override;
   // Return the reader associated with this subscriber
-  DDS::Entity * get() override { return reader_; }
+  DDS::Entity * get_entity() override { return reader_; }
 private:
   friend Raf;
-  DDSSubscriber(DDS::DomainParticipant_var dp, const rosidl_message_type_support_t & ros_ts,
-                const char * topic_name, const rmw_qos_profile_t & rmw_qos);
+  DDSSubscriber(DDS::DomainParticipant_var dp, const rosidl_message_type_support_t * ros_ts,
+                const char * topic_name, const rmw_qos_profile_t * rmw_qos);
   ~DDSSubscriber() { cleanup(); }
   void cleanup();
 
+  DDSTopic topic_;
```

Comment:
Is this used anywhere?

Reply:
Yes, topic_ is used. See DDSSubscriber.cpp.